ruby/ext/google/protobuf_c/repeated_field.c

```diff
@@ -419,19 +407,6 @@ VALUE RepeatedField_hash(VALUE _self) {
 
 /*
  * call-seq:
- *     RepeatedField.inspect => string
- *
- * Returns a string representing this repeated field's elements. It will be
- * formated as "[<element>, <element>, ...]", with each element's string
- * representation computed by its own #inspect method.
- */
-VALUE RepeatedField_inspect(VALUE _self) {
-  VALUE self_ary = RepeatedField_to_ary(_self);
-  return rb_funcall(self_ary, rb_intern("inspect"), 0);
-}
```

Comment:
easy method to remove and consolidate in ruby.  It is quite a bit slower in ruby but I figure it won't be used very often, so less code is better